runtime/centrifuge/src/lib.rs

```diff
@@ -62,7 +64,7 @@ use orml_traits::{currency::MutationHooks, parameter_type_with_key};
 use pallet_anchors::AnchorData;
 pub use pallet_balances::Call as BalancesCall;
 use pallet_collective::{EnsureMember, EnsureProportionAtLeast, EnsureProportionMoreThan};
-use pallet_ethereum::Transaction as EthereumTransaction;
+use pallet_ethereum::Transaction as EthTransaction;
```

Comment:
I would prefer that we just literally refer to it with the pull path instead of aliasing the import. That's also what we do on Altair. It's only referred to in 3 places so it's fine and we save the indirection.

Reply:
I did this because `EthereumTransaction` is how we reference the actual `pallet-ethereum-transaction` that we added.